packages/orchestrator/test_consensus_gate.py

```
@@ -0,0 +1,435 @@
+            proposal_type=ProposalType.CODE_CHANGE,
+            summary="x" * 201,
+            body="body",
+            blast_radius=BlastRadius.LOCAL,
+        ).validate()
+    except ValueError as e:
+        assert "E_CLAIM_INVALID_SCHEMA" in str(e)
+    else:
+        raise AssertionError("expected ValueError")
+
+
+def test_adr_writer_produces_file(tmp_path_factory=None):
```

Ruff (0.15.9)

[warning] Unused function argument: `tmp_path_factory`

(ARG001)